docs/summary.md

```diff
@@ -0,0 +1,11 @@
+# Code review bot for developers
+
+This documentation is targeted towards developers of the code review bot itself, or developers of analyzers who want to integrate it in our bot.
+
+Here is the overall summary of available documentation:
+
+- [How to trigger your task on new diffs](trigger.md)
+- [How to publish issues on Phabricator](publication.md)
+- [The default analysis format supported](analysis_format.md)
```

Comment:
I would link this third document from `publication.md` itself, since it's a subdocument of it basically.